app/controllers/articles_controller.rb

```diff
@@ -15,8 +15,8 @@ def new
 
   def create
     p = article_params
```

Comment:
avoid using `p` for variable names, since it means `print`. I'm surprised it even lets you do that!